manifests/kustomize/base/installs/generic/pipeline-install-config.yaml

```diff
@@ -12,8 +12,11 @@ data:
     namespace: `kubectl rollout restart deployment -n <your-namespace>`.
   appName: pipeline
   appVersion: 2.0.0
-  dbHost: mysql
-  dbPort: "3306"
+  dbHost: mysql  # relic to be removed after release
+  dbPort: "3306" # relic to be removed after release
+  dbType: mysql
+  mysqlHost: mysql
```

Comment:
Sorry, I'm not sure understand. Did you mean to name it something like `DBConfig.MySQLConfig.Port`?

Reply:
That is right, because it is a configmap, you can call it like `dbconfig_mysqlconfig_port`.

Reply:
I'm not sure about this. The rest of the variable names are not consistent with their counterparts in `config.json` either. If we change all of them, this might lead to more backward compatibility issues.